graph_logic/requirements/Fire Sanctuary.yaml

```
@@ -0,0 +1,242 @@
+hint-region: Fire Sanctuary
+
+Main:
+  First Room:
+    locations:
+      Hit Water Plant: Distance Activator | Bomb Bag
+    exits:
+      Main Exit: Nothing
+
+    Past Water Plant:
+      entrance: Hit Water Plant
+      locations:
+        First Room - Hit Water Plant: Distance Activator | Bomb Bag
+        Defeat Last Bokoblin: Can Defeat Bokoblins
+        Chest: Defeat Last Bokoblin
+      exits:
+        First Room: Hit Water Plant
+        Exit to Second Room: Fire Sanctuary Small Key
+
+  Second Room:
+    Outside Section:
+      exits:
+        Exit to First Room: Nothing
+        Second Room: Nothing
+
+    Balcony:
+      exits:
+        Second Room: Defeat Magmanos
+      locations:
+        Chest: Nothing
+
+    locations:
+      Chest: Nothing
+      Open Doors to Water Fruit: Mogma Mitts
+      Defeat Magmanos: Open Doors to Water Fruit & Sword
+
+    exits:
+      Outside Section: Nothing
+      Balcony: Defeat Magmanos
+      First Bridge: Nothing
+
+  First Bridge:
+    exits:
+      Second Room: Lizalfos Fight
+      Room with Water Plant: Lizalfos Fight
+      Second Bridge - Left: Second Bridge - Unlock Shortcut to First Bridge
+    locations:
+      Lizalfos Fight: Can Defeat Lizalfos
+
+  Room with Water Plant:
+    locations:
+      Blow up Rock: Hook Beetle
+      Hit Water Plant: Distance Activator | Bomb Bag
+    exits:
+      First Bridge: Nothing
+      Past Lava: Blow up Rock & Hit Water Plant
+
+    Past Lava:
+      locations:
+        Unlock Key Door: Fire Sanctuary Small Key x 2
+      exits:
+        Room with Water Plant: Projectile Item
+        First Trapped Mogma Room: Nothing
+        Water Fruit Room: Unlock Key Door
+
+  First Trapped Mogma Room:
+    locations:
+      Chest: Gust Bellows | Clawshots
+    exits:
+      Room with Water Plant - Past Lava: Nothing
+      Lower Area: Magmanos Fight Room - Magmanos Fight
+
+    Lower Area:
+      locations:
+        Reward: Nothing
+      exits:
+        First Trapped Mogma Room: Magmanos Fight Room - Magmanos Fight
+        Magmanos Fight Room - Upper Part: Magmanos Fight Room - Magmanos Fight
+        Magmanos Fight Room - Lower Part: Mogma Mitts
+
+  Water Fruit Room:
+    locations:
+      First Chest: Nothing
+      Second Chest: Nothing
+      Hydrate Frog: Sword
+      Room with Water Plant - Unlock Key Door:
+        Fire Sanctuary Small Key
+    exits:
+      Room with Water Plant - Past Lava:
+        Room with Water Plant - Unlock Key Door
+      After Frog: Hydrate Frog
+
+    After Frog:
+      exits:
+        Water Fruit Room: Hydrate Frog
+        Magmanos Fight Room - Upper Part: Nothing
+
+  Magmanos Fight Room:
+    Upper Part:
+      locations:
+        Magmanos Fight: Sword
+      exits:
+        Water Fruit Room - After Frog: Magmanos Fight
+        First Trapped Mogma Room: Magmanos Fight
+    Lower Part:
+      locations:
+        Unlock Key Door: Fire Sanctuary Small Key x 3
+        Move Sliding Door: Mogma Mitts
```

``
```suggestion
        Move Sliding Door: Mogma Mitts & Gust Bellows
```
You need Gust Bellows to blow away the ashes there